db/migrate/20210323180540_change_item_id_type_to_uuid.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ChangeItemIdTypeToUuid < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :versions, :item_id, :bigint
+    remove_index :versions, column: %i[item_type item_id], if_exists: true
+    add_column :versions, :item_id, :uuid, default: "gen_random_uuid()", null: false
```

Comment:
Not sure about the default here...

Reply:
The default was just to appease rubocop as otherwise the build was failing.  I don't believe we have any important data in that table anyway so I'm quite happy to erase it on dev